CMakeLists.txt

```diff
@@ -607,17 +607,20 @@ if(EL_TESTS)
       set_source_files_properties("${DRIVER}" PROPERTIES
         OBJECT_DEPENDS "${PREPARED_HEADERS}")
       target_link_libraries(tests-${TYPE}-${TESTNAME} El)
+      if(NOT NO_BINARY_SUBDIRECTORIES)
```

Comment:
I'll follow this up with a fix for this, because then I also need to update the spec file. In the interest of getting a build, I am going to merge.